packages/redis/src/lib/channel.ts

```diff
@@ -60,7 +58,7 @@ const cacheClient = createRedisConnection();
  * @param name name of the channel
  * @returns cache channel object
  */
-export const createCacheChannel = <TData>(name: string) => {
+export const createCacheChannel = <TData>(name: string, cacheTime: number = 5 * 60 * 1000) => {
```

Comment:
As I needed the caching channel stuff as well in my pi hole pr it might make sense if we decide which of the implementations we want. On mine I also added a `consumeAsync` function where you can provide a callback to fetch new data when the cache time has elapsed

Reply:
We can discuss some sort of final form together, decide on which features we need